### PR TITLE
MAGEDOC-3228: Fix broken links in the Installation Guide

### PR DESCRIPTION
**Pull request by jeff-matthews**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [x] Bug fix or improvement

## Summary

When this pull request is merged, it will fix broken links in the `install-gde/` directory.

### CODE REVIEW

**Comment by jeff-matthews** [comment]
@dshevtsov, are you saying I should remove the `/` until we update that variable? The link checker didn't flag that as broken.

**Comment by dshevtsov** [comment]
@jeff-matthews yes.
But if you say that this change doesn't introduce issues, we could leave it as is I think.
Maybe server recognizes double slashes and converts it into single slashes.

@belbiy what do you think?

File: guides/v2.1/install-gde/trouble/php/tshoot_xdebug.md [hunk 1]
```diff
@@ -23,4 +23,4 @@ During the installation, a  message similar to the following displays:
 ### Solution
 
 There is a known issue with <code>xdebug</code> that can affect Magento installations or access to the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %} or {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %} after installation.</p>
-<p>For details, see [Known issue with xdebug]({{ page.baseurl }}/install-gde/trouble/tshoot_install-issues.html#known-devbeta-xdebug).
+<p>For details, see [Known issue with xdebug]({{ page.baseurl }}/install-gde/trouble/tshoot_install-issues.html).
```
Comment (dshevtsov) [inline guides/v2.1/install-gde/trouble/php/tshoot_xdebug.md]:
```suggestion

For details, see [Known issue with xdebug]({{ page.baseurl }}/install-gde/trouble/tshoot_install-issues.html).
```

File: guides/v2.1/install-gde/trouble/php/tshoot_xdebug.md [hunk 2]
```diff
@@ -23,4 +23,4 @@ During the installation, a  message similar to the following displays:
 ### Solution
 
 There is a known issue with <code>xdebug</code> that can affect Magento installations or access to the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %} or {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %} after installation.</p>
```
Comment (dshevtsov) [inline guides/v2.1/install-gde/trouble/php/tshoot_xdebug.md]:
```suggestion
There is a known issue with <code>xdebug</code> that can affect Magento installations or access to the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %} or {% glossarytooltip 18b930cf-09cc-47c9-a5e5-905f86c43f81 %}Magento Admin{% endglossarytooltip %} after installation.
```

**Comment by jeff-matthews** [comment]
Let's leave it as-is. I want to merge these fixes and start running the link checker on the `install-gde/` dir right away. See #3228.

File: guides/v2.1/install-gde/trouble/php/tshoot_xdebug.md (hunk 1 above)
Comment (jeff-matthews) [reply guides/v2.1/install-gde/trouble/php/tshoot_xdebug.md]: No, that's out of scope for this PR.

Comment (jeff-matthews) [reply guides/v2.1/install-gde/trouble/php/tshoot_xdebug.md]: Actually, never mind, I'll do it real quick.

Comment (dshevtsov) [reply guides/v2.1/install-gde/trouble/php/tshoot_xdebug.md]: you can just accept the suggested change right on GitHub

Comment (dshevtsov) [reply guides/v2.1/install-gde/trouble/php/tshoot_xdebug.md]: If a line starts from HTML tag, Markdown would left unparsed. The similar issue like with Markdown in HTML tables.

**Pull request merged**